### PR TITLE
Tta #46 fix create cohort

### PR DESCRIPTION
**Pull request by deliailu27**

### CODE REVIEW

File: src/pages/createCohort/components/cohortList.jsx [hunk 1]
```diff
@@ -17,20 +24,67 @@ const CohortList = () => {
...
 
+  const updateCohortName=(event)=>{
+    event.preventDefault()
+    if (!newCohortName){setError(true); setTimeout(() => {
+      setError(false)
+    }, 3000);}
+    else{
+    client.patch(`/cohort/${event.target.id}`,{name:newCohortName},true).then(res=>{if (res.data.status === 'success') {
+      setUpdateCohortRes(true);
+    }}).catch(console.log)
```
Comment (vherus) [inline src/pages/createCohort/components/cohortList.jsx:35]: This is quite difficult to read, please format this a little nicer so it's easier to tell what the if statements and callback functions are related to

File: src/pages/createCohort/components/cohortList.jsx [hunk 2]
```diff
@@ -17,20 +24,67 @@ const CohortList = () => {
...
     </>
   );
 };
 export default CohortList;
+
+
+//<Card className="cohort-card" key={`${cohort.id}`}>
+             // {`cohort ${cohort.id} - ${cohort.name}`}
+             // <IconButton className='edit' onClick={updateCohortName}>< EditIcon/></IconButton>
+           // </Card>
```
Comment (vherus) [inline src/pages/createCohort/components/cohortList.jsx:90]: Best to remove dead code like this to keep a clean main branch

File: src/pages/createCohort/components/createCohortByName.jsx [hunk 3]
```diff
@@ -5,26 +5,32 @@ import './style.css';
...
   const [createCohortRes, setCreateCohortRes] = useState(false);
-  const [cohortName, setCohortName] = useState('');
+  const [name, setCohortName] = useState('');
 
   function createCohortName(event) {
     event.preventDefault();
     setCohortName(event.target.value);
+    
   }
+  console.log(name)
```
Comment (vherus) [inline src/pages/createCohort/components/createCohortByName.jsx:15]: Let's keep debug logs out of the main branch

File: src/pages/createCohort/components/createCohortByName.jsx [hunk 4]
```diff
@@ -5,26 +5,32 @@ import './style.css';
...
         if (res.data.status === 'success') {
           setCreateCohortRes(true);
+          
         }
       })
       .catch(console.log);
     setTimeout(() => {
       setCreateCohortRes(false);
     }, 3000);
+    window.location.reload()
```
Comment (vherus) [inline src/pages/createCohort/components/createCohortByName.jsx:33]: What's the reason for this reload?

**Review by vherus — CHANGES_REQUESTED** [review]

File: src/pages/createCohort/components/createCohortByName.jsx (hunk 4 above)
Comment (deliailu27) [reply src/pages/createCohort/components/createCohortByName.jsx:33]: to refresh the page so the new name would show

**Review by vherus — APPROVED** [review]

**Review by hubertlemczak — APPROVED** [review]

**Pull request merged**